README.rdoc

```diff
@@ -14,7 +14,7 @@ game programming conventions. Particularly it:
 * Right hand rule: 0 degrees is east, 90 is north, etc.
 
 These allow simple things like Trigonometry functions to work as
-expected. It means that all that stuff you were taught it grade school
+expected. It means that all that stuff you were taught in grade school
 still work as intended. This makes one less thing you have to adjust
```

Comment:
The grammar nerd in me can't tell if "work" needs to be "works" since "stuff" is singular but 🤷‍♀️

Reply:
Honestly I think "works" is proper since "stuff" is (I think?) a plural noun.

I especially like this:

> • British informal, dated worthless or foolish ideas, speech, or writing; rubbish: [as exclamation] : stuff and nonsense!